src/docs/GetStarted/Designers/PresetAdeo/index.mdx

```diff
@@ -0,0 +1,14 @@
+---
+title: 'Preset Adeo'
+order: 6
+---
+
+> Mozaic is designed to be above all the design system of the LEROY MERLIN brand. However, we also offer you the possibility to extend Mozaic to meet specific customization needs.
+
+## Adeo color palette
+
+It is in this context that we have set up the possibility of customizing Mozaic to meet the needs of LEROY MERLIN's parent company: ADEO.
```

Comment:
We have set Mozaic to be customisable to meet the needs of LEROY MERLIN's main brand: ADEO.